core/src/main/java/org/elasticsearch/indices/IndicesService.java

```diff
@@ -1129,8 +1129,4 @@ public void onRemoval(RemovalNotification<IndicesRequestCache.Key, IndicesReques
         (Index index, IndexSettings indexSettings) -> canDeleteIndexContents(index, indexSettings);
     private final IndexDeletionAllowedPredicate ALWAYS_TRUE = (Index index, IndexSettings indexSettings) -> true;
 
-    public IndexMetaData getIndexMetaData(Index index) {
```

Comment:
it will cause false positives but I don't feel strong enough to block this for such a nit pick